### PR TITLE
4 things in one

### PR DESCRIPTION
**Pull request by West-Tiwan**
loading screen ,contribution is react icon now ,sign-in and sign-up done ,add habit button in right place.

will rename classes after this merger

### CODE REVIEW

**Comment by anuragdaksh7** [comment]
image
this + icon is a bit too thick maybe try reducing the side size of + icon

**Comment by West-Tiwan** [comment]
> image this + icon is a bit too thick maybe try reducing the side size of + icon

will reduce scale to 1.1 during tonight's meet

**Comment by West-Tiwan** [comment]
> image this + icon is a bit too thick maybe try reducing the side size of + icon


Done

**Comment by anuragdaksh7** [comment]
image
image
cant see any change in size
+ resolve conflicts

**Comment by West-Tiwan** [comment]
> image image cant see any change in size


Check again

File: app/home/components/Habits/Habit/Habit.jsx [hunk 1]
```diff
@@ -30,7 +31,7 @@ const Habit = (props) => {
           </div>
         </div>
         <p onClick={handleSubmit} className={styles.addContribution}>
-          {habitFormIconsJson["plus"]}
+          <FaPlus/>
```
Comment (anuragdaksh7) [inline app/home/components/Habits/Habit/Habit.jsx]: <GoPlus />

File: app/home/components/Habits/Habit/Habit.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { habitFormIconsJson, themeColorJson } from "@/constants";
 // styles
 import styles from "./Habit.module.scss";
 import useHabit from "@/hooks/apis/useHabit";
+import {FaPlus} from "react-icons/fa6";
```
Comment (anuragdaksh7) [inline app/home/components/Habits/Habit/Habit.jsx]: import { GoPlus } from "react-icons/go";

**Comment by anuragdaksh7** [comment]
the test would fail coz main branch was failing

File: app/home/components/Habits/AllHabit/AllHabit.jsx [hunk 3]
```diff
@@ -42,6 +41,7 @@ const AllHabit = (props) => {
             );
           })}
       </div>
+
```
Comment (prashant-software-developer) [inline app/home/components/Habits/AllHabit/AllHabit.jsx]: remove extra line, use prettier formmater

File: app/home/home.module.scss [hunk 4]
```diff
@@ -1,6 +1,35 @@
 @import "@/constants/styles/variables.scss";
+@import "@/constants/styles/mixins.scss";
+
+.loading__page{
+  font-size: xxx-large;
```
Comment (prashant-software-developer) [inline app/home/home.module.scss]: $ use dollar before variable

File: app/home/home.module.scss [hunk 5]
```diff
@@ -1,6 +1,35 @@
...
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+  background-color: $gray-1000;
+}
 
 .page__wrapper {
-  padding-right: 10px;
-  padding-left: 10px;
+  height: 100vh;
```
Comment (prashant-software-developer) [inline app/home/home.module.scss:18]: do  not use vh this much time

File: app/sign-in/[[...sign-in]]/page.jsx [hunk 6]
```diff
@@ -1,5 +1,7 @@
 import { SignIn } from "@clerk/nextjs";
-
+import styles from "./sign-in.module.scss";
 export default function Page() {
```
Comment (prashant-software-developer) [inline app/sign-in/[[...sign-in]]/page.jsx:5]: one line space

File: app/sign-in/[[...sign-in]]/sign-in.module.scss [hunk 7]
```diff
@@ -0,0 +1,10 @@
+@import "@/constants/styles/variables.scss";
+
+.sign__in{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
```
Comment (prashant-software-developer) [inline app/sign-in/[[...sign-in]]/sign-in.module.scss]: vh -> do not use this much time

File: constants/styles/variables.scss [hunk 8]
```diff
@@ -1,7 +1,7 @@
 // colors/gray
-$gray-100: #ffffff;
+$gray-100: #e3e3e3;
```
Comment (prashant-software-developer) [inline constants/styles/variables.scss:2]: discuss once in meet

File: containers/Forms/CreateHabit.jsx [hunk 9]
```diff
@@ -159,10 +159,8 @@ export default function CreateHabit(props) {
           </div>
         </form>
       </div>
-      <button
-        className={!visible ? styles.visible__form : styles.hidden__form}
-        onClick={(e) => toggleVisible()}
-      >
+      <button className={visible ? styles.hidden__button : styles.visible__button}
+              onClick={(e) => toggleVisible()}>
```
Comment (prashant-software-developer) [inline containers/Forms/CreateHabit.jsx]: where are u using 'e', if not remove

**Review by prashant-software-developer — CHANGES_REQUESTED** [review]

File: app/home/home.module.scss (hunk 5 above)
Comment (West-Tiwan) [reply app/home/home.module.scss:18]: okieee

File: constants/styles/variables.scss (hunk 8 above)
Comment (West-Tiwan) [reply constants/styles/variables.scss:2]: ok

File: containers/Forms/CreateHabit.jsx (hunk 9 above)
Comment (West-Tiwan) [reply containers/Forms/CreateHabit.jsx]: that wasn't me , it was from main branch

**Review by prashant-software-developer — APPROVED** [review]

**Pull request merged**